docs/minikube.md

```diff
@@ -0,0 +1,43 @@
+# Deploying Antrea on Minikube
+
+<!-- toc -->
+- [Install Minikube](#install-minikube)
+  - [Deploy Antrea to Minikube cluster](#deploy-antrea-to-minikube-cluster)
+  - [Deploy a local build of Antrea to Minikube cluster (for developers)](#deploy-a-local-build-of-antrea-to-minikube-cluster-for-developers)
```

Comment:
"Deploy Antrea" shouldn't be a subsection of "Install Minikube". Two deploy options could be under one section that is in parallel with "Install Minikube" and "Verification", for example:

- Install Minikube
- Deploy Antrea
  - Deploy a released version to Minikube cluster
  - Deploy a local build to Minikube cluster
- Verification